lib/setup.php

```diff
@@ -336,6 +345,23 @@ function handle_wholesaler_message() {
 	update_post_meta( $post_id, 'contact_count', 0 );
 }, 10, 3 );
 
+/**
+ * Update wholesaler categories
+ */
+add_action( 'save_post', function( $post_id ) {
+	// Not the correct post type, bail out
+  if ( 'custom' !== get_post_type( $post_id ) ) return;
+  $post_categories = [];
+  $post_categories[] = get_field( 'category' )->term_id;
+  if ( get_field( 'minor_category_1' ) ) {
+    $post_categories[] = get_field( 'minor_category_1' )->term_id;
+  }
+  if ( get_field( 'minor_category_2' ) ) {
+    $post_categories[] = get_field( 'minor_category_2' )->term_id;
+  }
+  wp_set_post_terms( $post_id, $post_categories, 'customtaxonomy' );
+} );
+
```

Comment:
Tohle by možná v budoucnu stálo za to udělat genericky, ale jestli je v zadání jen 2, tak ok.